app/models/accounting/quickbooks/updater.rb

```diff
@@ -39,6 +39,7 @@ def update(loans = nil)
       #
       def qb_sync_for_loan_update
         raise NotConnectedError unless qb_connection
+        raise AccountsNotSelectedError unless Division.root.qb_accounts_selected?
```

Comment:
in `division.rb` the method `qb_division` seems to be set up to handle the qb connection being on a non-root division. Hopefully we can sort this out with some refactoring soon?